src/hooks/useActivityFeed/useActivityFeed.ts

```diff
@@ -43,7 +45,8 @@ const useActivityFeed = (
 
   useEffect(() => {
     setPageNumber(1);
-  }, [filters]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [JSON.stringify(filters, null, 2)]);
```

Comment:
Could you provide a comment about this?

Reply:
I tested something and forgot to undo the changes here.